sc/social/like/interfaces.py

```diff
@@ -202,3 +204,34 @@ class ISocialLikeSettings(model.Schema):
         required=False,
         default='',
     )
+
+    model.fieldset(
+        'open_graph',
+        label=u'Open Graph',
+        fields=[
+            'image_fallback',
+            'image_scale'
+        ],
+    )
+
+    form.widget('image_fallback', NamedImageFieldWidget)
+    image_fallback = schema.ASCII(
+        title=_(u'Image fallback'),
+        description=_(
+            u'help_image_fallback',
+            default=u'Content that does not have an image should show this image.',
+        ),
+        required=False,
+        # constraint=validate_image_settings,
+    )
+
+    image_scale = schema.Choice(
+        title=_(u'Image scale for shared'),
+        description=_(
+            u'help_image_scale',
+            default=u'',
+        ),
+        required=True,
+        default=u'large',
+        vocabulary='sc.social.likes.ImagesScales',
+    )
```

Comment:
this field looks like it don't belongs to this issue, should be addressed to the right issue and pull request

Reply:
Yes, this field was requested in redmine.